src/ledger.ts

```diff
@@ -1643,7 +1643,8 @@ export class LedgerV2SignMultisigTransaction extends LedgerBitcoinV2WithRegistra
   async signPsbt(): Promise<LedgerSignatures[]> {
     return this.withApp(async (app: AppClient) => {
       this.signatures = await app.signPsbt(
-        this.psbt,
+        // unchained-bitcoin mimics ledgers' methods
+        this.psbt as unknown as LedgerPsbtV2,
```

Comment:
I suppose this is working sometimes, but I don't know why. Could it be the cause of the other signature tests in Caravan failing? I believe unchained-bitcoin's `PsbtV2` only provides `serialize` as a common method. Is it possible other calls are being made to different methods which do not exist on `PsbtV2` and that is why the non-P2SH signatures are failing?

If not, this needs to probably have a more clear comment explaining that unchained-bitcoin PsbtV2 really only provides `serialize`.

Reply:
calls are being made to `copy`. I suppose there might be something different about P2SH PSBTs that allows the two versions to be in parity for those transactions and then break for segwit types.